app/views/header.html

```diff
@@ -9,6 +9,7 @@
 		<link rel="stylesheet" href="/public/assets/css/main.css" />
 		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
 		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
If possible please use the stylesheet locally, since the application would be deployed inside ITB network